globalbibletools/semantic-dictionary
# feat: script to extract the individual entries from a ubs dictionary xml file

What's changed:
- Created a nodejs script to extract the lexicon entries from a ubs dictionary xml file into individual files in the `data/${languageCode}` directory.

Testing:
1. Download an xml file from https://github.com/ubsicap/ubs-open-license/tree/main/dictionaries
2. Run the script file (`extract-ubs-dictionary.js`) with `node`, passing in the language code of the dictionary file and the location of dictionary file.
    - Example: `node extract-ubs-dictionary.js fr /path/to/downloaded/dictionary`
3. Navigate to the `data` folder. Confirm that there is a new folder for the language code. Confirm that this folder contains the lexicon entries of the downloaded dictionary.

Closes #1
